flask/sessions.py

```diff
@@ -84,21 +84,25 @@ class TaggedJSONSerializer(object):
     def dumps(self, value):
         return json.dumps(_tag(value), separators=(',', ':'))
 
+    LOADS_MAP = {
+        ' t': tuple,
+        ' u': uuid.UUID,
+        ' b': b64decode,
+        ' m': Markup,
+        ' d': parse_date
```

Comment:
little nitpick, please add a comma at the end, and  reindent the closing braces

that makes it more future friendly and a pleasure to read